go.mod

```diff
@@ -4,6 +4,6 @@ go 1.19
 
 require (
 	github.com/google/go-pkcs11 v0.2.0
-	golang.org/x/crypto v0.10.0
-	golang.org/x/sys v0.9.0
+	golang.org/x/crypto v0.11.0
+	golang.org/x/sys v0.10.0
```

Comment:
Hmm, this is weird, did you mean to update the dependencies?

Reply:
Hmm I don't recall purposely changing this so I'll change it back